sched/init/nx_start.c

```diff
@@ -376,7 +372,7 @@ void nx_start(void)
        * the IDLE task.
        */
 
-      g_idletcb[i].cmn.affinity = SCHED_ALL_CPUS;
+      g_idletcb[i].cmn.affinity = (cpu_set_t)CONFIG_SMP_DEFAULT_CPUSET;
```

Comment:
should we mask off the bits out of range?

Reply:
done